test/e2e/harness/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "jsx": "react",
+    "esModuleInterop": true,
```

Comment:
It would be nice to have this flag off. It does some weird things and can hide some errors for us. Many of our customers will expect everything to work with this flag off.

same for the next flag "allowSyntheticDefaultImports" (they are related)

Reply:
this file is auto-generated by `react-scripts`, I'll try if I can remove this flag.